packages/edit-site/src/components/left-sidebar/navigation-panel/index.js

```diff
@@ -88,9 +99,16 @@ const NavigationPanel = () => {
 						navigateToMenu="template-parts"
 					/>
 
+					<NavigationItem
+						title={ __( 'Content' ) }
+						navigateToMenu="content"
+					/>
+
 					<TemplatesMenu onActivateItem={ setTemplate } />
 
 					<TemplatePartsMenu onActivateItem={ setTemplatePart } />
+
+					<ContentMenu onChangePage={ setPage } />
```

Comment:
Should we use `onActivateItem` instead of `onChangePage` here to conform with menus above? If not, maybe `post` would make more sense, since it's a more general term.

Reply:
Reworked the whole thing. This is now irrelevant.